services/spar/src/Spar/Run.hs

```diff
@@ -70,11 +70,10 @@ initCassandra opts lgr = do
 -- logger
 
 mkLogger :: Opts -> IO Logger
-mkLogger opts = Log.new $ Log.defSettings
-  & Log.setLogLevel (toLevel $ saml opts ^. SAML.cfgLogLevel)
-  & Log.setOutput Log.StdOut
-  & Log.setFormat Nothing
-  & Log.setNetStrings (logNetStrings opts)
+mkLogger opts = Log.new $ Log.simpleDefSettings level netstr
```

Comment:
The resulting logger used to have `format = Nothing` and now it will have `format = iso8601UTC`.

I have no idea whether it will break anything or not.

Reply:
as a rule, you need to look at my git commit messages.  :)

Reply:
Oh, I never do because I don't expect anyone to put anything there. I'll keep in mind that there might be hints in commit messages.

Reply:
> the date format defaults to the value in `defSettings` if `Nothing`

Where does that happen? In https://gitlab.com/twittner/tinylog/blob/develop/src/System/Logger.hs#L126 I see that `Nothing` and `Just` are treated differently. `setFormat` doesn't do anything clever either: https://gitlab.com/twittner/tinylog/blob/develop/src/System/Logger/Settings.hs#L73.

Reply:
https://gitlab.com/fisx/tinylog/blob/settings-auxiliaries/src/System/Logger.hs#L194

it's a bit hard to find, i had to grep the entire package for `iso8601UTC`.

Reply:
(do you have a suggestion how to document this here?  i thought it's just relevant for the change, but not for the resulting code.)

Reply:
You pointed out correctly that `renderer` is not the only place where this is used.

I added a test case that establishes that for `iso8601UTC`, there is no difference between the two cases in https://gitlab.com/twittner/tinylog/blob/develop/src/System/Logger.hs#L126

I also [opened this issue](https://gitlab.com/twittner/tinylog/issues/2).

Reply:
I pulled your Gitlab branch and there is a difference:

```
> logger <- new (simpleDefSettings Debug False)
> fatal logger (msg "hi")
2019-03-05T23:04:24Z, F, hi

> logger2 <- new (setFormat Nothing $ simpleDefSettings Debug False)
> fatal logger2 (msg "hi")
F, hi
```

Reply:
hm, appearently i didn't add the test to Spec.hs :-(